docs/developer/process/agent-release/pre-release.md

```diff
@@ -20,19 +20,22 @@ Ensure that you have configured the following:
 
 ## Before Freeze
 
-1. Make a dependency update PR 1 week before freeze:
+1. _One week before freeze_: Make dependency update PRs:
     * Create a new branch
-    * Run `ddev dep updates --sync`
+    * Run `ddev dep updates --sync --check-python-classifiers --batch-size 10`
```

Comment:
> You may have to add the dependency to the set of [IGNORED_DEPS](https://github.com/DataDog/integrations-core/blob/master/datadog_checks_dev/datadog_checks/dev/tooling/commands/dep.py) and *revert that change.*

Does "revert that change" refer to reverting the dependency update change that is causing a CI failure?